network-api/networkapi/wagtailpages/factory/homepage_cause_statement_link.py

```diff
@@ -0,0 +1,16 @@
+from factory import (
+    Faker
+)
+from networkapi.utility.faker.helpers import reseed, get_homepage
+
+
+def generate(seed):
+    print('Generating Homepage Cause Statement Link')
+
+    home_page = get_homepage()
+
+    reseed(seed)
+
+    home_page.cause_statement_link_text = Faker('text', max_nb_chars=80)
```

Comment:
@KalobTaulien I need a hand here. I don't know why this creates `<factory.faker.Faker object at 0x7f9ed6152e90>` instead of the actual fake text. 🤔

Reply:
@mmmavis you can use `Faker('text', max_nb_chars=80).generate()` to generate the text instead of having it return the memory location. That should solve this.

Reply:
Updated! Thank you!